psakefile.ps1

```diff
@@ -0,0 +1,104 @@
+properties {
+  $config = Get-Content (Join-Path $PSScriptRoot 'package.json') | ConvertFrom-Json
+  $langaugeServerPath = (Join-Path $PSScriptRoot 'vendor/languageserver')
+  $languageServerZip = Join-Path $PSScriptRoot 'editor_services.zip'
+  $syntaxFilePath = Join-Path $PSScriptRoot 'syntaxes/puppet.tmLanguage'
+  $packageVersion = ''
+}
+
+task Clean {
+  $languageServerPath = (Join-Path $PSScriptRoot 'vendor')
+  if (Test-Path -Path $languageServerPath) {
+    Remove-Item -Path $languageServerPath -Recurse -Force
+  }
+  $syntaxPath = Join-Path $PSScriptRoot 'syntaxes/puppet.tmLanguage'
+  if (Test-Path -Path $syntaxPath) {
+    Remove-Item -Path $syntaxPath -Recurse -Force
+  }
+}
+
+task VendorEditorServices {
+  if ($config.editorComponents.editorServices.githubuser) {
+    $githubuser = $config.editorComponents.editorServices.githubuser
+  }
+  else {
+    $githubuser = 'lingua-pupuli'
+  }
+  if ($config.editorComponents.editorServices.githubrepo) {
+    $githubrepo = $config.editorComponents.editorServices.githubrepo
+  }
+  else {
+    $githubrepo = 'puppet-editor-services'
+  }
+
+  if ($config.editorComponents.editorServices.directory) {
+    throw "not implemented"
```

Comment:
Is this a TODO for the PR or will never do?

Reply:
was a TODO. will implement